kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -346,7 +414,14 @@ private String sanitizeTableName(String tableName) {
    *         name if sanitization is enabled.
    */
   public String getTableFromTopic(String topic) {
-    return getBoolean(SANITIZE_TOPICS_CONFIG) ? sanitizeTableName(topic) : topic;
+    String tableName = null;
+    Map<String, String> map = getTopicsToTables();
```

Comment:
What if we had the getTablesFromTopic method take in a tablesToTopics map as a parameter. In the BigQuerySinkTask we can store an instance of that map (since like you said getTablesToTopics is called on every single record) and pass that instance into the method. 

In the BigQuerySinkConfig and in the BigQuerySinkConnector, we can just call the method like so: `getTableFromTopic(topic, getTopicsToTables())`, instantiating the map on the spot. In these two classes I'm assuming it's okay to do this since I think the method will only be called once in each.